test/videotile/DefaultVideoElementResolutionMonitor.test.ts

```diff
@@ -70,5 +71,16 @@ describe('DefaultVideoElementResolutionMonitor', () => {
 
       monitor.removeObserver(mockObserver);
     });
+
+    it('should bind and unbind video elements', () => {
+      const videoElementFactory = new NoOpVideoElementFactory();
+      const videoElement = videoElementFactory.create();
+      expect(() => monitor.bindVideoElement(videoElement)).to.not.throw();
+      expect(() => monitor.bindVideoElement(null)).to.not.throw();
```

Comment:
I think it will good to expect `this.resizeObserver.observe()`  to havebeencalled(times) in both tests.

Reply:
Added boolean to represent if `observe` or `unobserve` is called in unit test and expect on them.